### PR TITLE
balabit/syslog-ng -> syslog-ng/syslog-ng transition

### PR DESCRIPTION
**Pull request by MrAnno**
- [x] Travis
- [x] Kira
- [x] Gitter
- [x] LGTM

Less important:
- [ ] Docker Hub
- [ ] Move ose-guides, syslog-ng-docker, syslog-ng-gitbook, syslog-ng-patterndb, other repos as well

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: .travis.yml [hunk 1]
```diff
@@ -112,7 +112,7 @@ matrix:
       sudo: required
       addons:
       before_install:
-      - docker pull balabit/syslog-ng-ubuntu-bionic
+      - docker pull syslog-ng/syslog-ng-ubuntu-bionic
```
Comment (szemere) [inline .travis.yml]: Unfortunately we do not have a syslog-ng namespace under DockerHub

**Comment by bazsi** [comment]
Iirc I registered syslogng without the dash

File: .travis.yml (hunk 1 above)
Comment (MrAnno) [reply .travis.yml]: Can we register that?

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by MrAnno** [comment]
@kira-syslogng : build packages please

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Package Build SUCCESS

Packages are available at https://ci.syslog-ng.com/jenkins/job/kira-dbld-pull-requests/613/artifact/

**Comment by MrAnno** [comment]
Removed the WIP flag. I renamed the GitHub/Travis/Gitter-related references, I guess it's enough for now.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -127,5 +127,5 @@ testing.
  [ar:gitter]: https://gitter.im/syslog-ng/syslog-ng
  [ar:mailing-list]: http://lists.balabit.com/mailman/listinfo/syslog-ng
  [ar:issue-tracker]: https://github.com/syslog-ng/syslog-ng/issues
- [ar:issues:help]: https://github.com/syslog-ng/syslog-ng/labels/help
```
Comment (Kokan) [inline CONTRIBUTING.md]: Please also update the text `help`, it is confusing this way.

Comment (MrAnno) [reply CONTRIBUTING.md]: Uhh, thanks!

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**